climakitae/tools/indices.py

```diff
@@ -47,7 +47,9 @@ def effective_temp(T):
 
 def noaa_heat_index(T, RH):
     """Compute the NOAA Heat Index.
-    See references for more information on the derivation on this index.
+    The heat index is a metric designed to quantify how hot air feels to the human body,
```

Comment:
```suggestion
    The Heat Index is a metric designed to quantify how hot air feels to the human body,
```
Heat Index is a proper noun